images/trust-manager/tests/02-helm-install.sh

```diff
@@ -1,26 +1,7 @@
 #!/usr/bin/env bash
 
-# monopod:tag:k8s
-
 set -o errexit -o nounset -o errtrace -o pipefail -x
 
-function preflight() {
-	if [[ "${IMAGE_REGISTRY}" == "" ]]; then
-		echo "Must set IMAGE_REGISTRY environment variable. Exiting."
-		exit 1
-	fi
-
-	if [[ "${IMAGE_REPOSITORY}" == "" ]]; then
-		echo "Must set IMAGE_REPOSITORY environment variable. Exiting."
-		exit 1
-	fi
-
-	if [[ "${IMAGE_TAG}" == "" ]]; then
-		echo "Must set IMAGE_TAG environment variable. Exiting."
-		exit 1
-	fi
-}
-
 function ensure_trust_manager(){
```

Comment:
We can just do `helm install --wait` and drop the sleep and `kubectl wait`.

Up to you, I can come back and get this later too.

Reply:
I'll try it now, and if it's not easy we can circle back to it